lib/engine/config/game/g_18_cz.rb

```diff
@@ -0,0 +1,1537 @@
+                    "abilities": []
+                },
+                {
+                    "name": "Nezamislitz – Morkowitz",
+                    "value": 75,
+                    "revenue": 20,
+                    "sym": "NM",
+                    "abilities": []
+                },
+                {
+                    "name": "Taus – Tachau",
+                    "value": 80,
+                    "revenue": 20,
+                    "sym": "TT",
+                    "abilities": []
+                }
+            ],
+            "corporations": [
+                {
+                    "float_percent": 50,
+                    "sym": "SX",
+                    "name": "Sächsische Eisenbahn",
+                    "logo": "18_cz/SX",
+                    "max_ownership_percent": 60,
+                    "tokens": [
+                        0,
+                        40
+                    ],
+                    "coordinates": [
+                        "A7",
+                        "B4"
+                    ],
+                    "color": "red",
+                    "type": "large"
+                },
+                {
+                    "float_percent": 50,
+                    "sym": "PR",
+                    "name": "Preußische Eisenbahn",
+                    "logo": "18_cz/PR",
+                    "max_ownership_percent": 60,
+                    "tokens": [
+                        0,
+                        40
+                    ],
+                    "coordinates": [
+                        "A21",
+                        "B18"
+                    ],
+                    "color": "black",
+                    "type": "large"
+                },
+                {
+                    "float_percent": 50,
+                    "sym": "BY",
+                    "name": "Bayrische Staatsbahn",
+                    "logo": "18_cz/BY",
+                    "max_ownership_percent": 60,
+                    "tokens": [
+                        0,
+                        40
+                    ],
+                    "coordinates": [
+                        "F2",
+                        "H4"
+                    ],
+                    "color": "blue",
+                    "type": "large"
+                },
+                {
+                    "float_percent": 50,
+                    "sym": "kk",
+                    "name": "kk Staatsbahn",
+                    "logo": "18_cz/kk",
+                    "max_ownership_percent": 60,
+                    "tokens": [
+                        0,
+                        40
+                    ],
+                    "coordinates": [
+                        "J14",
+                        "I17"
+                    ],
+                    "color": "orange",
+                    "type": "large"
+                },
+                {
+                    "float_percent": 50,
+                    "sym": "Ug",
+                    "name": "ungar. Staatsbahn",
```

Comment:
seems like it's called `Ungarische Staatsbahn` in the rules, why is it different here?